@xen-orchestra/fs/src/abstract.js

```diff
@@ -25,6 +25,10 @@ type RemoteInfo = { used?: number, size?: number }
 type File = FileDescriptor | string
 
 const checksumFile = file => file + '.checksum'
+const formatSpeed = (hrtime: Array, size: Number) => {
+  const seconds = (hrtime[0] + hrtime[1] / 1e9).toFixed(4)
```

Comment:
Rounding or truncating values before display is a bad idea, especially inside a computation.